python/ray/dag/dag_node_operation.py

```diff
@@ -122,14 +176,38 @@ def __hash__(self):
         """
         return hash((self.operation, self.task_idx))
 
+    def __str__(self):
+        class_name = (
+            self.actor_handle._ray_actor_creation_function_descriptor.class_name
+        )
+        actor_id = self.actor_handle._actor_id.hex()
+        return (
+            class_name
+            + "_"
+            + actor_id[:4]
+            + f" [{self.operation.exec_task_idx}] "
+            + f"{self.operation.method_name} {self.operation.type}"
+        )
+
+    def _get_actor_id(self):
+        return self.actor_handle._ray_actor_id.hex()
```

Comment:
Nit: maybe use `_actor_id` instead? `_actor_id` is a property, which seems more suitable for external access https://github.com/ray-project/ray/blob/master/python/ray/actor.py#L1557.